lib/ddtrace/propagation/distributed_headers/b3.rb

```diff
@@ -0,0 +1,46 @@
+require 'ddtrace/ext/distributed'
+require 'ddtrace/propagation/distributed_headers/base'
+require 'ddtrace/propagation/distributed_headers/headers'
+
+module Datadog
+  module DistributedHeaders
+    # B3 provides helpers to inject or extract headers for B3 style headers
+    class B3
+      include Ext::DistributedTracing
+
+      class << self
+        include Base
```

Comment:
I think you can replace this with `extend Base`.

Reply:
I tried that, and it didn't seem to work. I can try again, just in case I was doing something off before.